docs/guide/worker.md

```diff
@@ -43,6 +43,57 @@ Worker starting in daemon mode with `queue/listen` command supports [File], [Db]
 [Beanstalk]: driver-beanstalk.md
 [Gearman]: driver-gearman.md
 
+Systemd
+-------
+
+Systemd is a Linux system manager used to bootstrap daemons. To configure workers startup
+using systemd, create a config file named `yii-queue@.service` in `/etc/systemd/system` with
+the following contents:
+
+```conf
+[Unit]
+Description=Yii Queue Worker %I
+After=network.target
+After=mysql.service
+Requires=mysql.service
+
+[Service]
+User=www-data
+Group=www-data
+ExecStart=/usr/bin/php /var/www/my_project/yii queue/listen --verbose
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+```
+
+You need to reload systemd in order for it to re-read configuration:
+
+```sh
+systemctl daemon-reload
+```
+
+Set of commands to control workers:
+
+```sh
+# To start two workers
+systemctl start yii-queue@1 yii-queue@2
+
+# To get status of running workers
+systemctl status "yii-queue@*"
+
+# To stop the worker
+systemctl stop yii-queue@2
+
+# To stop all running workers
+systemctl stop "yii-queue@*"
+
+# To start two workers at system boot
+systemctl enable yii-queue@1 yii-queue@2
+```
+
+To learn all features of systemd, check its documentation.
```

Comment:
a link to the documentation should be added here.

Reply:
https://freedesktop.org/wiki/Software/systemd/#manualsanddocumentationforusersandadministrators